kernel/sched.c

```diff
@@ -292,7 +292,13 @@ _SYSCALL_HANDLER(k_thread_priority_set, thread_p, prio)
 	_SYSCALL_OBJ(thread, K_OBJ_THREAD);
 	_SYSCALL_VERIFY_MSG(_VALID_PRIO(prio, NULL),
 			    "invalid thread priority %d", (int)prio);
-	_SYSCALL_VERIFY_MSG(prio >= thread->base.prio,
+	/*
+	 * All arguments comes from userspace has data type int or u32_t
```

Comment:
this is a very wordy comment, I don't think it's necessary.
it's not quite correct either, all arguments are u32_t, full stop.

Reply:
@andrewboie  Done